.github/workflows/pr-builder.yml

```
@@ -19,10 +19,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)